.buildkite/pipeline.yml

```diff
@@ -31,6 +31,20 @@ steps:
       - github_commit_status:
           context: Prototype Build
 
+  - group: ":rocket: Prototype Build"
+    if: build.branch == trunk
+    steps:
+      - input: Deploy Prototype Build
+        prompt: Share a Prototype Build via Firebase App Distribution?
+        key: prototype_triggered
+      - label: Prototype Build
+        depends: prototype_triggered
+        command: .buildkite/commands/prototype-build.sh
+        plugins: [$CI_TOOLKIT]
+        notify:
+          - github_commit_status:
+              context: Prototype Build From Trunk
```

Comment:
Notice the different name for the check, just in case the same commit already run from a PR (impossible with merge commits and squashes, but one never knows... rebase and merge? cherry pick and force push bypassing checks?) and also to keep things tidy.

Reply:
Ah good point, completely forgot about that UI in GitHub to show checks on any commit even outside PRs 👍